### PR TITLE
Cleaning up canary

### PR DESCRIPTION
**Pull request by sjelin**
Just a couple of minor things

### CODE REVIEW

File: client/main.js [hunk 1]
```diff
@@ -18,10 +18,10 @@ var socket = io.connect('http://' + location.host, {
 // instantiate the updater of the view
 new StatusUpdater(socket, util.elm('title'), util.elm('banner'), util.elm('browsers'));
 window.karma = new Karma(socket, util.elm('context'), window.open,
-	window.navigator, window.location);
+    window.navigator, window.location);
 
-// TODO(julie): is there a better way to store this?
```
Comment (sjelin) [inline client/main.js]: I don't think we're going to think of a better way of allowing the test to find the socket.  For instance, vojta's old way was super hacky.  This is at least explicit.

**Comment by sjelin** [comment]
@juliemr donno if you saw this

**Comment by juliemr** [comment]
Looks solid. Let's wait to merge until we make sure that the current canary release is OK.

**Comment by Dignifiedquire** [comment]
LGTM

**Comment by Dignifiedquire** [comment]
Closing due to various github issues.

**Pull request closed without merging**